redux-form/lib/Field.d.ts

```diff
@@ -202,12 +203,12 @@ interface WrappedFieldInputProps {
     /**
      * A function to call when the form field loses focus.
      */
-    onBlur: FocusEventHandler<any>;
+    onBlur: (eventOrValue: FocusEvent<any> | FieldValue) => void;
 
     /**
      * A function to call when the form field is changed.
      */
-    onChange: FormEventHandler<any>;
+    onChange: (eventOrValue: FormEvent<any> | FieldValue) => void;
```

Comment:
Also `newValue` and `previousValue` parameters are missing. 

`onFocus`,  `onDragStart`, and `onDrop` props are missing as well.